Tests/GravatarUITests/AvatarPickerViewModelTests.swift

```diff
@@ -64,6 +68,103 @@ final class AvatarPickerViewModelTests {
         }
     }
 
+    @Test
+    func testFetchOriginalSizeAvatarSuccess() async throws {
+        await model.refresh()
+        guard let avatar = model.grid.avatars.first else {
+            #expect(Bool(false), "No avatar found")
+            return
+        }
```

Comment:
This is a great place to use the `#require()` macro.  It is like `#expect()`, except that if it fails, the test returns immediately.  It's primary use case is unwrapping optionals.

```suggestion
        let avatar = try #require(model.grid.avatars.first, "No avatar found")
```

Reply:
Great idea! :TIL: 👍